lib/vanagon/engine/pooler.rb

```diff
@@ -37,7 +37,7 @@ def select_target
           '{"' + @platform.vmpooler_template + '":"1"}',
           { 'X-AUTH-TOKEN' => @token }
         )
-        if response and response["ok"]
+        if response["ok"]
```

Comment:
http_request could return an empty hash, but will never return nil. http_request would raise an error in the nil case. indexing into a non-existent key on a hash returns nil.